### ISSUE TITLE
Get Asset Components

### ISSUE THREAD

**Issue by kovrobert** [barrycarey/SnipeSharp#20]
Hello Barry,

i would like to ask, how can i get all of the components of an asset? I searched upon a property in the Asset and the Components class but i was no able to find a matching one. Can you give me any feedback upon this issue?
This is important because i have many csv files, what i have to work with and if i can't check the component on an asset i will create it more then it should. Since i got new csv files every day and a service do the job for me, this will be a big problem because in a year i will have to much trash in the database.

Thank You for you're help:
Robi

**Comment by Christoffleroux** [comment]
Yes please, the api has no way to associate a component with an asset
